Documentation/k8s-pre-reqs.md

```diff
@@ -18,6 +18,8 @@ setting up Rook in a Kubernetes cluster with Pod Security Policies enabled.
 
 ## Flexvolume Configuration
 
+**NOTE** This configuration is only needed when using the FlexVolume driver (required for Kubernetes 1.12 or earlier). The Ceph-CSI RBD driver is recommended for Kubernetes 1.13 and newer, making FlexVolume configuration redundant.  
```

Comment:
FlexVolume driver wasn't just used for mounting RBD, but also for mounting CephFS.
@leseb Should CephFS be added here as well? What do you think?

Reply:
Seems legit. @ranrubin can you also add that to your PR? Thanks!